### PR TITLE
`entrypoint-wrapper`: exit with wrapped exit code

### PR DESCRIPTION
**Pull request by Prucek**
This PR changes the exit code of the entrypoint-wrapper to return the exit code of the wrapped process.

cc @bbguimaraes 
[DPTP-3533](https://issues.redhat.com/browse/DPTP-3533) does not need this, because the wrapper as of now still returns zero if success, just when the child exits with another code it will exit with 1. That's changed here. 

- [x] Can be merged after #3560 is merged.

### CODE REVIEW

**Review by bbguimaraes — COMMENTED** [review]
Looks good, just a few implementation notes.

File: cmd/entrypoint-wrapper/main.go [hunk 1]
```diff
@@ -62,12 +63,13 @@ func main() {
...
-		os.Exit(1)
+		os.Exit(errorCode)
 	}
-	if err := opt.run(); err != nil {
+	if code, err := opt.run(); err != nil {
 		fmt.Fprintln(os.Stderr, "error:", err)
-		os.Exit(1)
+		os.Exit(code)
 	}
+	os.Exit(0)
```
Comment (bbguimaraes) [inline cmd/entrypoint-wrapper/main.go:72]: This line is not necessary, simply returning from `main` has the same effect.

**Review by bbguimaraes — COMMENTED** [review]
/test e2e
/lgtm

**Comment by Prucek** [comment]
Sorry, I forgot, this PR cannot be merged before #3560

**Pull request merged**